src/lib/steps.ts

```diff
@@ -48,6 +48,11 @@ export type DomainStep = {
   domain: string;
 };
 
+export type DuplicateColumnStep = {
+  name: 'duplicate';
+  column: string;
```

Comment:
Shouldn't we, for consistency-sake with other steps, add a `new_column?: string;` field?

Reply:
@adimasci it was on purpose: we wanted, as for the `delete` step, to get the column creation immediately effective after clicking, without having to fill in any form field. It is inspired from other softwares and looks lighter, offering a more direct feedback to the user.

But after reflexion, I think that you're right and that in our context we should ask for the new name in a form at creation, because a duplicated column will need a rename in 95% of the cases.
And it would be way heavier than other softwares I had in mind if we have to log an additional `rename` step in the pipeline every time we duplicate a column !

So I will amend this ! Thank you